DEVELOPMENT.md

```diff
@@ -0,0 +1,13 @@
+# Development
+
+This document describes how to get the opensourcestories.org site running locally.
+
+## Getting started
```

Comment:
Would you mind adding a little bit of information or links on using development containers? Thinking of at least a link to here: https://code.visualstudio.com/docs/remote/containers

This was my first time trying them out, and I hit some stumbling blocks :-)